src/hello.rs

```diff
@@ -0,0 +1,273 @@
+use std::{
```

Comment:
This file is largely the same, but git's diff algorithm couldn't detect it as a rename. There are some minor changes, but it's mostly just search and replace.